blog/content/posts/2025-11-25-first-mcp-anniversary.md

```diff
@@ -0,0 +1,160 @@
+
+But adoption can only grow as long as there are MCP servers to actually use, and clients which are capable of communicating with them. Within the same timeframe, we saw the number of active MCP servers go from just a few experimental ones to _thousands_. If you think about a scenario, it's likely there's an MCP server for it.
+
+Here are just a few of many (very many) MCP servers that you can try:
+
+- Notion [built an MCP server](https://github.com/makenotion/notion-mcp-server) to help you manage your notes.
+- Stripe has a [pretty extensive MCP server](https://docs.stripe.com/mcp#tools) to manage all kinds of payment workflows.
+- GitHub [built their own MCP server](https://github.com/github/github-mcp-server) to help developers automate their engineering processes.
+- Hugging Face [created an MCP server](https://github.com/huggingface/hf-mcp-server) to make model management and dataset search a breeze.
+- Postman [built their MCP server](https://github.com/postmanlabs/postman-mcp-server) to help automate API testing workflows.
+
+And there's so much more to discover in the MCP ecosystem! That's why we also launched the [MCP Registry](https://blog.modelcontextprotocol.io/posts/2025-09-08-mcp-registry-preview/) earlier this year. It's the central index for all available MCP servers that now has close to two thousand entries since its announcement in September. That's a 407% growth from the initial batch of servers we onboarded that same month.
+
+The ecosystem is growing, adoption is growing, but what's underpinning all of this?
+
+### Community & Governance
+
+MCP's growth was never a one‑company effort. Students, hobbyists, startup engineers, and enterprise architects all shaped the protocol - submitting SEPs, shipping SDKs in new languages, and stress‑testing some of the early assumptions we had about MCP in production. MCP went from an experiment to a standard that is used to connect LLMs to both APIs as well as applications (yes, there’s even a [Blender MCP server](https://github.com/ahujasid/blender-mcp)). That kind of organic adoption isn't something you can just come up with, no matter how ambitious your aspirations are with an open source project.
+
+![David Soria Parra presenting on MCP in May of 2025](/posts/images/first-mcp-anniversary/david-mcp-may.webp)
+
+It's all about the **MCP community**. Our community rallied around the protocol, organizing events like [MCP Dev Summit](https://mcpdevsummit.ai/), [MCP Night](https://workos.com/mcp-night), [MCP Dev Days](https://developer.microsoft.com/en-us/reactor/series/S-1563/), and showing up at marquee events like [AI Engineer World's Fair](https://wf2025.ai.engineer/worldsfair/2025) to share what they learned and built.
```

Comment:
Do we have any photos of a large group of people at one of these events? The current photos in the article don't give a sense of scale / how big and buzzing the community is in my opinion